reltime/reltime.py

```diff
@@ -118,6 +130,48 @@
 # ---- HELPER FUNCTIONS ---- #
 
 
+def split_hour_minute(time):
+    """Split hour/minute into components."""
+    stripped = time.strip().rstrip('?:!.,;')
+    hourminute = stripped.split(':')
+    if len(hourminute) > 1:
+        hour = int(hourminute[0])
+        minute = int(hourminute[1])
+    else:
+        hour = int(hourminute[0])
+        minute = 0
+    return hour, minute
+
+
+def split_am_pm(time):
+    """Separate time from am/pm if present."""
+    split_am = re.split(r'a', time, flags=re.IGNORECASE)
+    split_pm = re.split(r'p', time, flags=re.IGNORECASE)
+    if len(split_am) > 1:
+        time = split_am[0].strip()
+        am_pm = "am"
+    elif len(split_pm) > 1:
+        time = split_pm[0].strip()
+        am_pm = "pm"
+    else:
```

Comment:
Nope. I'm looking for one of three cases, 1. Letter "a" present, 2. letter "p" present, 3. neither present. If 1, use am, if 2 use pm, otherwise use default.